Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
OK this is from https://github.com/dockerfile/java/blob/master/oracle-java8/Dockerfile

It looks like Docker, inc has no idea about licenses and trademarks!!! This is crazy.